dev/test-spark-autologging.sh

```diff
@@ -21,9 +21,6 @@ pip install -e spark-3.0.0-preview-bin-hadoop2.7/python
 popd
 
 export SPARK_HOME=$TEMPDIR/spark-3.0.0-preview-bin-hadoop2.7
-find tests/spark_autologging -name 'test*.py' | xargs -L 1 pytest --large
+find tests/spark_autologging/datasource -name 'test*.py' | xargs -L 1 pytest --large
```

Comment:
This script doesn't have to run tests under `tests/spark_autologging/ml` since `dev/run-python-flavor-tests.sh` runs them.